index.html

```diff
@@ -368,14 +370,14 @@ <h3>What is a Verifiable Credential?</h3>
 Dutch passport, an American driving license, or a health insurance card)
           </li>
           <li>
-Information related to specific properties being asserted by
+Information related to specific properties asserted by
 the issuing authority about the [=subject=] (for example, nationality,
 the classes of vehicle entitled to drive, or date of birth)
           </li>
           <li>
-Evidence related to how the qualifications required for the [=credential=]
-to be issued were met by the [=subject=] (for example, a measurement,
-proof of citizenship, or test result)
+Evidence demonstrating how the [=subject=] met the qualifications required
+for issuing the [=credential=] (e.g., a measurement, proof of citizenship,
+or test result)
```

Comment:
```suggestion
Evidence demonstrating how the [=subject=] met the qualifications required
for issuing the [=credential=] (for example, a measurement, proof of 
citizenship, or test result)
```

We avoid the use of latin abbreviations (at least, in this spec) because many non-primary English readers don't know latin. "id est" and "exempli gratia". 

As a personal rule, I try to tone down the language for the lowest possible reading comprehension level. We fail at that throughout the spec many times, but lowering the reading comprehension level while staying accurate ensures that a wider audience can understand the spec (hopefully).